### DISCUSSION TITLE
Replacement Barometer

### DISCUSSION THREAD

**Discussion by DrewBrandt** [Terrapin-Rocket-Team/SRAD-Avionics#107] [Ideas]
Since the BMP390 we are using has issues with EMI, I'd like to investigate switching to a new one. I found [the MS5611](https://www.digikey.com/en/products/detail/te-connectivity-measurement-specialties/MS561101BA03-50/5277638) from a European company, which allegedly has a high refresh rate, is accurate to 10cm (4in), and is "immune" to EMI. It is twice as expensive though.

Thoughts?

**Comment by varun-un** [comment]
# [DPS310](https://www.infineon.com/dgdl/Infineon-DPS310-DataSheet-v01_02-EN.pdf?fileId=5546d462576f34750157750826c42242)

### Pros
- Pretty accurate +/- 2cm
- Temperature invariant

### Cons
- +/- 8m absolute accuracy

**Comment by varun-un** [comment]
# MS5611
